app/controllers/users/sessions_controller.rb

```diff
@@ -1,13 +1,22 @@
 class Users::SessionsController < Devise::SessionsController
   respond_to :json
```

Comment:
if we use API mode, do we need to specify respond_to ?

Reply:
No, in API mode we don't need it.
Fixed in commit "[commit-29] Remove unnecessary code"